content/tutorials/custom-zk-chain/20.customizing-your-chain.md

```diff
@@ -0,0 +1,360 @@
+
+First, shut down the node server running for `zk_chain_1`.
+
+Move back into the `my_elastic_chain` ecosystem folder and run the `chain create` command:
+
+```bash
+zk_inception chain create
+```
+
+This time, use the answers below:
+
+```bash
+$ zk_inception chain create
+
+┌   ZKsync toolbox
+│
+◇  What do you want to name the chain?
+│  custom_zk_chain
+│
+◇  What's the chain id?
+│  272
+│
+◇  Select how do you want to create the wallet
+│  Localhost
+│
+◇  Select the prover mode
+│  NoProofs
+│
+◇  Select the commit data generator mode
+│  Rollup
+│
+◇  Select the base token to use
+│  Custom
+│
+◇  What is the token address?
+│  0x_YOUR_TOKEN_ADDRESS
+│
+◇  What is the base token price nominator?
+│  1
+│
+◇  What is the base token price denominator?
+│  1
+│
+◇  Set this chain as default?
+│  Yes
+```
+
+The base token nominator and denominator are used to define the relation of the base token price with ETH.
+For example, if we set the nominator to 20 and the denominator to 1, together the relation is 20/1,
+this would mean that 20 tokens would be given the equivalent value as 1 ETH for gas.
+For testing purposes, we'll just use a 1:1 ratio.
+
+Next, initialize the chain in the ecosystem with:
+
+```bash
+zk_inception chain init
+```
+
+This time select no for the option to deploy a paymaster contract.
+Use the default answers for all other prompts.
+
+Now that the chain is initialized, you can start the chain server:
+
+```bash
+zk_inception server
+```
+
+## Bridging the base token to your chain
+
+The last step is to bridge some ERC20 tokens from the L1 to the new chain.
+Base tokens can not be minted on the L2 without being backed by the corresponding L1 amount. In the future, there will be more flexibility for this.
+
+Open a new terminal and use the `zksync-cli` to bridge the tokens with the command below:
+
+```bash
+npx zksync-cli bridge deposit --token 0x_YOUR_TOKEN_ADDRESS --rpc=http://localhost:3050 --l1-rpc=http://localhost:8545
+```
+
+```shell
+? Amount to deposit 5
+? Private key of the sender 0x7726827caac94a7f9e1b160f7ea819f172f7b6f9d2a97f992c38edeab82d4110 
+? Recipient address on L2 0x36615Cf349d7F6344891B1e7CA7C72883F5dc049
+```
+
+To verify that this worked, let's check the new balance of our address on the L2 chain:
+
+```bash
+zksync-cli wallet balance --address 0x36615Cf349d7F6344891B1e7CA7C72883F5dc049 --rpc http://localhost:3050
+```
+
+Note that we aren't checking for any specific token address here.
+The CLI will display the amount as ETH, but in reality, it's our bridged ERC20 tokens.
+
+And that's it! You are now running a local ZK chain that uses a custom ERC20 base token.
+You can follow the steps in the previous section to deploy a contract to the new chain.
+
+## Bridging ETH to your chain
+
+If your ERC20 token bridged to the L2 "looks" like ETH, then what happens if you bridge ETH to your custom chain?
```

Comment:
Maybe
```suggestion
If your ERC20 token bridged to the L2 "acts" like ETH, then what happens if you bridge ETH to your custom chain?
```

Not sure whichever is better, using "looks" makes me thing that this is still in relation to the earlier comment about zksync-cli display it as ETH.